yt_dlp/extractor/kick.py

```diff
@@ -117,12 +143,100 @@ def _real_extract(self, url):
                 response, ('livestream', ('session_title', 'slug')), get_all=False, default=''),
             'description': traverse_obj(response, ('livestream', 'channel', 'user', 'bio')),
             'channel': traverse_obj(response, ('livestream', 'channel', 'slug')),
-            'channel_id': str_or_none(traverse_obj(response, ('livestream', 'channel', 'id'))),
+            'channel_id': int_or_none(traverse_obj(response, ('livestream', 'channel', 'id'))),
             'uploader': traverse_obj(response, ('livestream', 'channel', 'user', 'username')),
-            'uploader_id': str_or_none(traverse_obj(response, ('livestream', 'channel', 'user_id'))),
+            'uploader_id': int_or_none(traverse_obj(response, ('livestream', 'channel', 'user_id'))),
             'timestamp': unified_timestamp(response.get('created_at')),
             'duration': float_or_none(traverse_obj(response, ('livestream', 'duration')), scale=1000),
             'thumbnail': traverse_obj(
                 response, ('livestream', 'thumbnail'), expected_type=url_or_none),
             'categories': traverse_obj(response, ('livestream', 'categories', ..., 'name')),
         }
+
+
+class KickClipIE(KickBaseIE):
+    IE_NAME = 'kick:clips'
+    _VALID_URL = r'https?://(?:www\.)?kick\.com/(?P<channel>[\w-]+)\?clip=(?P<id>clip_[\w-]+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?kick\.com/(?P<channel>[\w-]+)/?\?(?:[^#]+&)?clip=(?P<id>clip_[\w-]+)'
```

Reply:
or if we don't need `channel_slug`:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?kick\.com/[\w-]+/?\?(?:[^#]+&)?clip=(?P<id>clip_[\w-]+)'
```